packages/engine/Source/Renderer/Context.js

```diff
@@ -1340,7 +1340,14 @@ function continueDraw(context, drawCommand, shaderProgram, uniformMap) {
 
   if (defined(indexBuffer)) {
     offset = offset * indexBuffer.bytesPerIndex; // offset in vertices to offset in bytes
-    count = defaultValue(count, indexBuffer.numberOfIndices);
+    if (count) {
+      count =
```

Comment:
`Math.min` might be more readable than a ternary expression here:

```javascript
count = Math.min(count, indexBuffer.numberOfIndices);
```

Reply:
Fixed.